src/library_glut.js

```diff
@@ -199,6 +199,39 @@ var LibraryGLUT = {
       }
     },
 
+    touchHandler: function(event) {
+      if (event.target != Module['canvas']) {
+        return;
+      }
+
+      var touches = event.changedTouches,
+          main = touches[0],
+          type = "";
+
+      switch(event.type) {
+          case "touchstart":
```

Comment:
only 2 spaces indentation in this, and please use `{}` for the case bodies (or put the `type = X; break;` on the same line, which might be nicer, same for the return on default).